### PR TITLE
Add bandage_ng

### PR DESCRIPTION
**Pull request by neoformit**
Add fork of bandage as bandage_ng. This fork of bandage has new features and is actively maintained, while the original bandage is stale.

----

* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/bandage_ng/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "BandageNG" %}
```
Comment (abretaud) [inline recipes/bandage_ng/meta.yaml:1]: I don't know if you've seen, linting complains about the name of the folder which is not matching the name in the recipe: https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=23915&view=logs&j=eb1ca712-fec5-5d93-deb0-40983fe61f2f&t=c9ebc308-e058-58af-a4f4-866fa7649dbb&l=27
you'll have to choose between bandage_ng and bandageng :)

Comment (neoformit) [reply recipes/bandage_ng/meta.yaml]: Thanks for that, just fixed :+1:

**Comment by neoformit** [comment]
@abretaud :grin:

**Comment by abretaud** [comment]
Oh damn I wasn't fast enough to merge it after merging master, just remerged now :)

**Comment by abretaud** [comment]
@BiocondaBot please merge

**Review by abretaud — APPROVED** [review]

**Comment by abretaud** [comment]
@BiocondaBot please merge

**Comment by abretaud** [comment]
@BiocondaBot please merge

**Pull request merged**